action.yml

```
@@ -47,6 +47,14 @@ inputs:
   NO_GIT_PUSH:
     description: This is a private variable that is used by the maintainers of this Action for debugging.  Setting this value to true will prevent any changes from being saved to your repository.
     require: false
+  REPO2DOCKER_EXTRA_ARGS:
+    description: Extra commandline arguments to be passed to repo2docker
+    require: false
+  FORCE_REPO2DOCKER_VERSION:
+    description: Force a specific version of repo2docker to be installed. Either in the form of `repo2docker==<version>` for install from pypi, or `git+https://<github-url>@<commit-hash` for installing from a github repo / branch
+    require: false
+  EXTRA_PIP_INSTALL:
+    description: Extra packages to install in the image before running repo
```

Do we want this to be a required flag?
```suggestion
    description: Extra packages to install in the image before running repo
    require: false
```

---

I figure "in the image" is confusing as "the image" could be the GitHub action container image or the built repo2docker image. So I suggest "in the environment where repo2docker is run" instead.

But also since I suggested removing `FORCE_REPO2DOCKER_VERSION` in favor of sticking with `EXTRA_PIP_INSTALL` I figure we also need a note about using this to pin repo2docker's version.

And finally, this includes @GeorgianaElena `require: false` addition.

```suggestion
    description: Extra packages to install in the environment that runs repo2docker. You can for example use this to pin a specific version of repo2docker by providing `repo2docker==<version>` or `git+https://<github-url>@<commit-hash` to reference a repo2docker fork not on PyPI.
    require: false
```